src/main/webapp/vendor/toast/js/script.js

```diff
@@ -0,0 +1,17 @@
+function init() {
```

Comment:
This code seems like a simple instantiation of the library, so I think we can simply add this to our own non-vendor js file. There's nothing here that I'm concerned about for copyright/license issues since you're providing all your own options.

Reply:
Yes for now, but I know that more code will be added in the future that will need the license attached.

Reply:
What kind of code are we talking about? Can we split the code so that as much as possible, the code you write is in our own `main.js` or something, and you can import library functions from the vendored file?

Reply:
Yes, I can do that.